src/test/cpp/blaze_util_posix_test.cc

```diff
@@ -16,14 +16,14 @@
 #include <sys/resource.h>
 #include <sys/wait.h>
 
-#include <inttypes.h>
 #include <string.h>
 #include <unistd.h>
 
 #include <cerrno>
 #include <cstdarg>
 #include <cstdio>
 #include <cstdlib>
+#include <cstdint>
```

Comment:
Please keep includes sorted alphabetically. This has to come before cstdio.

Reply:
Done.